cassandra/util.py

```diff
@@ -1810,4 +1810,4 @@ def maybe_add_timeout_to_query(stmt: str, metadata_request_timeout: Optional[dat
     ms = int(metadata_request_timeout / datetime.timedelta(milliseconds=1))
     if ms == 0:
         return stmt
-    return f"{stmt} USING TIMEOUT {ms}ms"
+    return str(f"{stmt} USING TIMEOUT {ms}ms")
```

Comment:
> cassandra/util.py:1813:11: Cannot convert Unicode string to 'str' implicitly. This is not portable and requires explicit encoding.

Could you explain how is this not portable? It was my understanding that in Python 3 normal strings are unicode.

Reply:
https://cython.readthedocs.io/en/latest/src/tutorial/strings.html

Sounds like we are not using the correct language level, I don't think this should have with python 3

Reply:
> Sounds like we are not using the correct language level

Then we should do this instead of adding `str` call.

Reply:
I've found the error:

```python
def a() -> str:
  a = 10
  return f'{a}'
```

new cython consider now the type annotation

this works
```python
def a() -> unicode:
  a = 10
  return f'{a}'
```

Reply:
But why does it need `unicode`? `str` in python3 is unicode.

Reply:
`3` vs `3str` 
https://stackoverflow.com/a/57415689
(gist of it, 3str is for some python2 comparability)

sine we don't define the language level, it's now default to `3str`, with older version of Cython we used before it default to `2`

when running my example with `cython temp.py -3` no need to change the annotation

hence we should be picking the  language level correctly
(or adapt the code)

Reply:
How to we set the language level?

Reply:
https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#how-to-set-directives

Reply:
@fruch , thanks, I have updated setup.py instead.